test/stepDefinitions/searchResultSteps.js

```diff
@@ -0,0 +1,120 @@
+
+let SearchResultPage = require('../pageObjects/searchResultPage.js');
+let TestData = require('../utils/TestData.js');
+let chai = require("chai").use(require("chai-as-promised"));
+let baseSteps = require('./baseSteps.js');
+let expect = chai.expect;
+let Data = require('../utils/TestData.js');
+
+var { defineSupportCode } = require("cucumber");
+
+defineSupportCode(function ({ Given, When, Then, Before, After }) {
+
+  let searchResultPage = new SearchResultPage();
+  const valueArr = ['Odin', 'Freya', 'Demeter', 'Quetzcuatl', 'Thor', 'Zeus', 'Wotan', 'Ares', 'Zephyr', 'Baldur', 'Aegir', 'Heimdall', 'Loki', 'Tyr', 'Hera', 'Dionysus', 'Poseidon', 'Mithras', 'Khantengri', 'Ishtar', 'Ahuramazda', 'Aphrodite', 'Crom', 'Lady of the Lake', 'Morgana', 'Baphomet'];
+
+  async function bulkUpdateCases(arr) {
```

Comment:
this looks like it could be a useful method so perhaps we should move it to baseSteps.js ?